client/src/components/Button/BaseButton.js

```diff
@@ -67,6 +67,7 @@ const BaseButton = styled(Button)`
       pointer-events: none;
     `}
 
+
```

Comment:
remove this extra line

Reply:
Okay

Reply:
oh nooo, now this is merged into master. It's fine though. but please remove extra white lines next time.